tesseract/tesseract_scene_graph/include/tesseract_scene_graph/joint.h

```diff
@@ -239,7 +244,7 @@ class Joint
 
   void clear()
   {
-    this->axis.setZero();
+    this->axis = Eigen::Vector3d(1, 0, 0);
```

Comment:
I just saw that it was in the urdf spec

Reply:
> I'd think the most common case would be revolute joint with a rotation about the z.

can you explain why you say/write that?

For people with experience with DH, this may be true, but in ROS, X+ is the 'default' axis for just about everything (see [REP 103](https://www.ros.org/reps/rep-0103.html#axis-orientation)), so `x` being the default axis would seem to be correct.

Reply:
Yes, I was thinking of DH.